backend/roles/team-roles-manager.js

```diff
@@ -2,45 +2,62 @@
 
 const twitchApi = require("../twitch-api/api");
 const frontendCommunicator = require("../common/frontend-communicator");
+const accountAccess = require("../common/account-access");
 
 /**
- * @param {import('twitch/lib/API/Kraken/Team/Team').Team[]} teams
+ * @typedef MappedTeamRole
+ * @property {string} id
+ * @property {string} name
  */
-function mapRoles(teams) {
+
+/** @returns {MappedTeamRole} */
+const mapRoles = (teams) => {
     return teams
         .map(team => {
             return {
-                id: team.id,
+                id: parseInt(team.id), // For compatibility (Kraken API returned an int)
                 name: team.displayName
             };
         });
-}
-
-async function getAllTeamRolesForViewer(username) {
-    const roles = await twitchApi.teams.getMatchingTeamsByName(username);
+};
```

Comment:
Great thinking on making sure this remains compatible. 

Super minor, but we can strip away some unneeded syntax like so
```js
const mapRoles = (teams) => teams.map(t => ({
  id: parseInt(t.id),
  name: t.displayName
})
```

Reply:
Fixed.